pkg/providers/amifamily/bootstrap/mime/mime.go

```diff
@@ -0,0 +1,120 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mime
```

Comment:
Do we have/need testing for this package? I know that we can probably assume that this is technically correct since it was written and used already by the AL2 AMI. Mainly curious if this is fully covered by our launch template testing or if there is some coverage that we are missing that we should specifically functionally test for this package.

Reply:
On my list of things to add 👍